apiserver/facades/client/bundle/bundle_test.go

```diff
@@ -222,3 +234,16 @@ func (s *bundleSuite) TestGetChangesBundleEndpointBindingsSuccess(c *gc.C) {
 		}
 	}
 }
+
+func (s *bundleSuite) TestExportBundleSuccess(c *gc.C) {
+	_, err := s.auth.HasPermission(permission.ReadAccess, s.modelTag)
```

Comment:
This is code from the actual implementation - no need to do that here.
If you wanted to test permissions, you'd use a different user name when creating the fake authoriser.

Reply:
done